crates/test/Cargo.toml

```diff
@@ -2,6 +2,10 @@
 name = "trdelnik-test"
 version = "0.1.0"
 edition = "2018"
+authors = ["AckeeBlockchain <hello@ackeeblockchain.com>>"]
+license = "Apache-2.0"
+repository = "https://github.com/Ackee-Blockchain/trdelnik"
+description = "Trdelnik proc-macro"
```

Comment:
:thinking: i suggest to improve the description(s) :smile: 

think about SEO :muscle: from my point of view .. everybody should be able to search for it without knowing the name